src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -201,7 +203,7 @@ static Slice build(
                 );
                 break;
             case "php":
-                slice = new PhpComposer(cfg.path(), cfg.storage());
+                slice = new PhpComposer(cfg.storage());
```

Comment:
@olenagerasimova PHP slice needs to be wrapped into `trimIfNotStandalone`, so it will be:
```
slice = trimIfNotStandalone(
  settings, standalone,
  new PhpComposer(cfg.storage())
);
```
Same goes for NPM slice